filecoin-proofs/tests/api.rs

```diff
@@ -2121,3 +2132,99 @@ fn test_aggregate_proof_encode_decode() -> Result<()> {
 
     Ok(())
 }
+
+fn test_sealing_encode_decode_ranges<R: Rng, Tree: MerkleTreeTrait>(
+    rng: &mut R,
+    sector_size: u64,
+    pc1_output: &SealPreCommitPhase1Output<Tree>,
+    replica_path: &Path,
+) -> Result<()> {
+    let sector_nodes = (sector_size >> 5) as usize;
+    let node_range_len = 11;
+
+    for _ in 0..5 {
```

Comment:
Those are arbitrary values, aren't they? I'd put in a comment that they are arbitrary.